telemetry/definitions/commonDefinitions.json

```
@@ -2609,50 +2594,80 @@
             ],
             "passive": true
         },
+
         {
-            "name": "awsq_serviceInvocation",
-            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
-            "metadata": [
-                {
-                    "type": "awsqAssignClickCount"
-                },
-                {
-                    "type": "awsqGenerateCodeIterationCount"
-                },
-                {
-                    "type": "awsqGenerateCodeClickCount"
-                },
-                {
-                    "type": "awsqApproachIterationCount"
-                },
-                {
-                    "type": "awsqIsRegeneratedCode"
-                },
-                {
-                    "type": "awsqFilesReviewedCount"
-                },
-                {
-                    "type": "awsqIsAcceptedCodeChanges"
-                },
-                {
-                    "type": "awsqRepositorySize"
-                },
-                {
-                    "type": "awsqFilesChangedCount"
-                },
-                {
-                    "type": "awsqThumbsUpCount"
-                },
-                {
-                    "type": "awsqThumbsDownCount"
-                },
-                {
-                    "type": "awsqIsApproachAccepted"
-                },
-                {
-                    "type": "awsqServiceErrorsCount"
-                }
-            ]
+            "name": "awsq_assignCommand",
+            "description": "Number of times users have triggered /assign",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqAssignCommandCount" }]
+        },
+        {
+            "name": "awsq_codeGenerationIterations",
```

metrics should roughly map to "user actions". what action does this represent? if it's "code generation", then name the metric as such.
```suggestion
            "name": "awsq_generateCode",
```